packages/framework/composer.json

```diff
@@ -58,7 +58,7 @@
         "fzaninotto/faker": "^1.7.1",
         "gedmo/doctrine-extensions": "^2.4.34",
         "guzzlehttp/guzzle": "^6.3",
-        "helios-ag/fm-elfinder-bundle": "^10.0.4",
+        "helios-ag/fm-elfinder-bundle": "^10.1",
```

Comment:
I do not thing it is good to require nex minor of this package in our patch version. I may cause project to fail during `composer update`